backend/services/main-service/utils/packed_data/comments.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "commentId": 1,
+    "postId": 101,
+    "itineraryId": 201,
```

Comment:
The ItineraryId and PostId **must** have the same value, always.
The reason why is because whenever a Post is created, an Itinerary is created right after it.
Similarly, when either the Post/Itinerary are deleted, the corresponding Itinerary/Post will be deleted as well, meaning that the Ids will be the same, always